zephyr/adapter/src/wpa_supplicant/port/os_xtensa.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * wpa_supplicant/hostapd / Internal implementation of OS specific functions
+ * Copyright (c) 2005-2006, Jouni Malinen <j@w1.fi>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ *
+ * Alternatively, this software may be distributed under the terms of BSD
+ * license.
+ *
+ * See README and COPYING for more details.
+ *
+ * This file is an example of operating system specific  wrapper functions.
+ * This version implements many of the functions internally, so it can be used
+ * to fill in missing functions from the target system C libraries.
+ *
+ * Some of the functions are using standard C library calls in order to keep
+ * this file in working condition to allow the functions to be tested on a
+ * Linux target. Please note that OS_NO_C_LIB_DEFINES needs to be defined for
+ * this file to work correctly. Note that these implementations are only
+ * examples and are not optimized for speed.
+ */
+
+#include "os.h"
+#include <stdlib.h>
+#include "esp_system.h"
+#include "utils/common.h"
+#include <random/rand32.h>
+
+int os_get_time(struct os_time *t)
+{
+	if (t == NULL) {
```

Comment:
@sylvioalves Is there coding style problem here (tab alignment)? Please check once.